Docs/Configuration.tex

```diff
@@ -575,31 +572,31 @@ \subsection{Installation and Upgrade}\label{configuration-install}
 the \hyperref[miscsecurityprops]{Security Properties} section of this document.
 
 The \texttt{OC\ config} file, as with any property list file, can be edited with
-any text editor such as nano and vim. However, specialised software may provide
-a better experience. On macOS, the preferred GUI application is
+any text editor, such as TextEdit or NotePad. However, specialised software
```

Comment:
(Two more quick things, sorry: It's Notepad, not NotePad - assuming it's the Windows one - also I don't know the current status, but historically Notepad was not great with line endings.... just thought of this, might matter?)

Reply:
Will fix thanks.

Yes, it might matter if indeed the case.

Not an expert on Notepad ... just assumed it must be on par with TextEdit but that might turn out to be a bit of a leap of faith.

Reply:
No, not at all, if you want an equivalent, you'd have to choose Notepad++.

But I don't believe "any editor" needs changing, "any (non-borked) editor" should go without saying!

Reply:
Best not to leave things to interpretation but will leave in place.
When everything gets "done" additional passes will be required.

Reply:
Best to just restore "nano or vim" and close the chapter.

Reply:
Yes, editing with TextEdit makes little sense to me. If you have UI, you could use Xcode. Please restore.

Reply:
Was already restored as per my previous but just a note that your interpretation is not quite the sense that is transmitted. What is transmitted is that you can use any basic editor but specialised tools (which Xcode is) can give a better experience.

However, this is trivial in the scheme of things.  Was already restored as stated.